### ISSUE TITLE
Kiwix refuses to read old zim file

### ISSUE THREAD

**Issue by lands39** [kiwix/kiwix-desktop#1186]
Re-opening issue 1173. I was on the latest version and it still refuses to work. i have a recording of the issue: https://files.catbox.moe/hygo8m.mp4

the reason i want this old file is to look at some older information. this file refuses to work both on windows and on the server version of kiwix. 

using version 2.3.1-2 for the desktop viewer and version 3.3.0-3+b1 of kiwix-tools on debian.

**Comment by kelson42** [comment]
@mgautierfr I'm sure it works with dev version, but we still miss the windows nightly. Can you please fix and update?

**Comment by lands39** [comment]
https://files.catbox.moe/bs57wg.mp4 recording of the issue on kiwix-serve, version 3.3.0-3+b1 on debian. the older file doesn't show up whatsoever, however, multiple up to date copies of wikipedia will show up and work no problem.

**Comment by kelson42** [comment]
@lands39 If you have acces to a linux, have you tested latest nightly http://download.kiwix.org/nightly/2024-08-22/ ? Both kiwix-desktoo and kiwix-tools?

**Comment by lands39** [comment]
i started playing around and now the old zim file shows up on kiwix-serve, its very broken though and none of the URLs work. if you hit the die button, you can get sent to a random article in the file so there's something on the file.

**Comment by kelson42** [comment]
@lands39 Yes, URL are absolute in the HTML and we don't support this anymore fully.

**Comment by kelson42** [comment]
@lands39 What should I do with this issue, to me everything is clear.

**Comment by lands39** [comment]
don't know, it simply doesn't open in the desktop reader. what do you mean by clear?

**Comment by kelson42** [comment]
@lands39 It works with dev versions of both kiwix-serve and kiwix-desktop. You can use the nightlies or just wait the releases. Regarding URL in the content, there is nothing we can do, we have changed the specification around 10 years agonand this file is even older.
